docs/sdk_users/running_examples.md

```
@@ -2214,3 +2214,7 @@ record = (
 print(f"Generated PRNG bytes length: {len(record.prng_bytes)} bytes")
 print(f"PRNG bytes in hex: {record.prng_bytes.hex()}")
 ```
+
+## See Also
+
+- [Error Handling](error_handling.md) — how to catch and handle `PrecheckError`, `ReceiptStatusError`, and `MaxAttemptsError`
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)